.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Feature Request"
```

Comment:
I worry that the perceived amount of work here may dissuade someone from filling out a feature request. Wdyt?